### ISSUE TITLE
Cannot enable extensions

### ISSUE THREAD

**Issue by btlechowski** [brave/browser-laptop#13838]
### Steps to Reproduce

  1. Clean Install
  2. Open about:preferences#extensions
  3. Enable an extension

**Actual result:**
Extension cannot be enabled. It is stack on version 'Latest'
![image](https://user-images.githubusercontent.com/34715963/38820533-7a643aea-419e-11e8-94a1-14a8863a5864.png)

**Expected result:**
Possible to enable extensions

**Reproduces how often:**
100%

### Brave Version
 
```
Brave: 0.22.663 
V8: 6.6.346.23 
rev: b35e0b516ef183171397f87f81fbbe2449e859c7 
Muon: 5.2.2 
OS Release: 6.1.7601 
Update Channel: Release 
OS Architecture: x64 
OS Platform: Microsoft Windows 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 66.0.3359.106
```

cc @LaurenWags @kjozwiak @srirambv @darkdh

**Comment by darkdh** [comment]
fixed in muon 5.2.3 by https://github.com/brave/muon/commit/d4effccfda02d2e35cd1a3d6d68a59a9e14d9a1f

**Comment by btlechowski** [comment]
Verified Ubuntu 17.10
Version 0.22.664
Muon 5.2.4
